Samples.xcodeproj/project.pbxproj

```diff
@@ -271,7 +272,6 @@
 		108EC04029D25B2C000F35D0 /* QueryFeatureTableView.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = QueryFeatureTableView.swift; sourceTree = "<group>"; };
 		1C0C1C3429D34DAE005C8B24 /* ChangeViewpointView.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = ChangeViewpointView.swift; sourceTree = "<group>"; };
 		1C42E04329D2396B004FC4BE /* ShowPopupView.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = ShowPopupView.swift; sourceTree = "<group>"; };
-		1C965C4729DBA8CD002F8536 /* Bristol */ = {isa = PBXFileReference; lastKnownFileType = folder; name = Bristol; path = ../../../../../Desktop/Bristol; sourceTree = "<group>"; };
```

Comment:
A few things to point out about the ODR. You are almost there!

- Be sure to check the "path" on the right pane to make sure the resource is pointing to a relative path to the project, not to a local directory
- This plane model is a special case: it also requires the 2 jpg to render without warning. So cannot only add the plane dae model, we need to add the whole folder as a "Folder Reference", instead of a "Group".

Reply:
I added the ODR as a folder reference instead of a group and updated the tag name. I'm not sure if the ODR path is correct or not.

Reply:
Sorry I didn't make myself clear. My commit already fixed all the issues. Please revert the changes in the project file.

Reply:
I do not see the ODR data locally so that is why I added it again.
<img width="467" alt="Screen Shot 2023-04-06 at 4 05 57 PM" src="https://user-images.githubusercontent.com/117859673/230509530-78dd8e09-9b64-402b-b1e3-6fab59265b40.png">

Reply:
Can you try to build the project so the `Download Portal Item Data` phase can execute and download the data?

If it doesn't work, can you try to remove all content in the Portal Data folder and build the project, so it does a clean download?

You can also take a look at the build log to see if there is any anomaly.

Reply:
How should I use `DowloadPortalItemData.swift` under `Scripts`? I added the initial data manually. Did you add the script and arguments to the Samples Scheme so that the script runs with each build?

Reply:
The script doesn't need to be run manually, it is part of the build phases.

My workflow is in `wiki/iOS-On-Demand-Resources`. I suggested the workaround in Swift 2261 because I thought you might need to tweak the code with the data before making the commits. 😬 

Basically, for a sample that needs ODR, what I do are as follows

- Create the sample folder, add README and metadata.
- Build and run the project, so the script in the build phase will download the data derived from the portal item ID in the metadata.
- Add the downloaded data to the project, either as a group or a folder reference depending on the folder structure requirements for each data
- Write the code.

<details><summary>Advanced method</summary>

Sometimes when there is only 1 data file, I create a dummy file in the portal ID folder, add it to the project, then delete the folder from disk. Therefore Xcode project file has that reference created, but shows as red (file missing on disk). And when the script runs in the build phase, it will fill the missing data.

This is not recommended unless you are exactly sure where everything needs to go in the project.

</details>

Reply:
p.s., if you see a crash on the file path line of code on the first clean run, that might be related to #48 😬 I haven't figured out the root cause yet. The second run should be smooth.

Reply:
Running the project did not download the data for me. 
<img width="462" alt="Screen Shot 2023-04-07 at 8 35 34 AM" src="https://user-images.githubusercontent.com/117859673/230646037-8680b8dc-e093-40cf-b9f5-f3afb983abe9.png">